lib/spack/spack/stage.py

```diff
@@ -357,12 +357,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def expected_archive_files(self):
         """Possible archive file paths."""
         paths = []
-
         fnames = []
-        expanded = True
```

Comment:
I'm not sure that we actually want this change? Might have been reverted by accident when switching away from CurlFetchStrategy?